src/oic/oic/__init__.py

```diff
@@ -1818,11 +1818,15 @@ def make_id_token(self, session, loa="2", issuer="",
         return idt
 
 
-def scope2claims(scopes):
+def scope2claims(scopes, extra_scope_dict=None):
```

Comment:
Looking at the method, it is surprising it doesn't simply return a set() and instead creates this dict with all values empty. Not sure if any caller uses the values in a meaningful way...

Reply:
The method is used in only one place so I guess the answer is no :)

Looking at the code it is a legacy thing...

If you want, we can make a ticket for refactoring of this thing...